dev-haskell/heterocephalus/heterocephalus-1.0.5.7.ebuild

```diff
@@ -16,15 +16,17 @@ LICENSE="MIT"
 SLOT="0/${PV}"
 KEYWORDS="~amd64 ~x86"
 
-RDEPEND=">=dev-haskell/blaze-html-0.9:=[profile?]
-	>=dev-haskell/blaze-markup-0.8:=[profile?]
-	>=dev-haskell/dlist-0.8:=[profile?]
-	>=dev-haskell/shakespeare-2.0:=[profile?]
-	>=dev-haskell/template-haskell-compat-v0208-0.1.2:=[profile?]
-	>=dev-lang/ghc-8.4.3:=
+RDEPEND=">=dev-haskell/blaze-html-0.9.1.2:=[profile?]
+	>=dev-haskell/blaze-markup-0.8.2.8:=[profile?]
+	>=dev-haskell/dlist-1.0:=[profile?]
+	>=dev-haskell/parsec-3.1.14.0:=[profile?]
+	>=dev-haskell/shakespeare-2.0.26:=[profile?]
+	>=dev-haskell/template-haskell-compat-v0208-0.1.7:=[profile?]
+	>=dev-haskell/text-1.2.4.1:=[profile?]
+	>=dev-lang/ghc-9.0.2:=
```

Comment:
I just want to point out that you're removing the only version that will work with `ghc-8.10`. However, I haven't seen any evidence anyone still uses those old versions of GHC and it might be time to sunset them, so I think this is fine.